### ISSUE TITLE
Joe & Mac 2 freezes at this part

### ISSUE THREAD

**Issue by themalu866** [snes9xgit/snes9x#660]
Playing Joe & Mac 2 - Lost in the Tropics on Windows 7, and no, I'm not using any cheats at all.
If you touch the door on the right the game will freeze but it only happens WHILE riding the dinosaur, music still plays but you can't do anything.
Here is a save state and a screenshot of the moment.
![yty565](https://user-images.githubusercontent.com/42208976/91052781-56416d80-e5f8-11ea-9f05-2b7a97175800.png)
[Joe & Mac 2 - Lost in the Tropics.zip](https://github.com/snes9xgit/snes9x/files/5118267/Joe.Mac.2.-.Lost.in.the.Tropics.zip)

**Comment by Eggo-dotcom** [comment]
same happens here.

**Comment by Eggo-dotcom** [comment]
This issue is almost a year old, not normal around here. Why is no one commenting? Its a legit bug.

**Comment by bearoso** [comment]
I tried tracing it a while ago. I never found the exact cause, but it's another timing issue. 

I haven't worked on Snes9x recently, and this is an annoying one. Fixing timing things like this usually means another one pops up because of the fix. If I try to fix it completely correctly, people complain, so it's a game of workaround tetris, trying to find a bunch that fit together.

If you want to play the game at the moment, I suggest bsnes or ares.

**Comment by Eggo-dotcom** [comment]
I see, i just saw no replies so thought he went un noticed. Thanks for looking at it.

**Comment by kscorpionx** [comment]
It also happens on real hardware, i used everdrive and real snes console

**Comment by Irata-sys** [comment]
> It also happens on real hardware, i used everdrive and real snes console

So its not a snes9x bug its the actual game?

**Comment by kscorpionx** [comment]
> > It also happens on real hardware, i used everdrive and real snes console
> 
> So its not a snes9x bug its the actual game?

Yes, it is not, this also happens in other emulators like bsnes.
